src/mongo/db/ldap/ldap_manager_impl.cpp

```diff
@@ -537,7 +537,9 @@ static void init_ldap_timeout(timeval* tv) {
     tv->tv_usec = (timeout % 1000) * 1000;
 }
 
-Status LDAPManagerImpl::execQuery(std::string& ldapurl, std::vector<std::string>& results) {
+Status LDAPManagerImpl::execQuery(std::string& ldapurl,
```

Comment:
As far as I can see, the `ldapurl` variable is read-only in the context of the `execQuery` function. Does it make sense to change its type to `const std::string&`?

Reply:
const specifier added